author: Andres Salomon <dilinger@debian.org>

We don't have adler2 yet:
cp: cannot stat 'default_for_rust_host_build_tools/obj/build/rust/std/libadler2.rlib': No such file or directory

--- a/build/rust/std/BUILD.gn
+++ b/build/rust/std/BUILD.gn
@@ -50,7 +50,7 @@
     # These are no longer present in the Windows toolchain.
     stdlib_files += [
       "addr2line",
-      "adler2",
+      "adler",
       "gimli",
       "libc",
       "memchr",
